views/aggregate-block-metadata.json

```diff
@@ -1655,15 +1655,15 @@
             "read-path",
             "write-path"
           ],
-          "version": "2.19.2"
+          "version": "2.19.4"
         }
       },
       "string": {
         "name": "String",
         "slug": "string",
         "logo_url": "https://cdn.sanity.io/images/3ugk85nk/production/c262ea2c80a2c043564e8763f3370c3db5a6b3e6-48x48.png",
         "documentation_url": "https://docs.prefect.io/api-ref/prefect/blocks/system/#prefect.blocks.system.String",
-        "description": "A block that represents a string. Deprecated, please use Variables to store string data instead.",
```

Comment:
@collincchoy this message should still be here right?

Reply:
I'm not sure. Maybe @WillRaphaelson or @zangell44?

Reply:
Yes!